_data/package.yml

```diff
@@ -79,28 +79,20 @@ keywords: |
     transformation semigroups, partial permutations, inverse semigroups, Green's relations, free inverse semigroup, partition monoid, bipartitions, Rees matrix semigroups.
 citeas: |
     <p class='BibEntry'>
-    [<span class='BibKey'>Mit18</span>]   <b class='BibAuthor'>Mitchell, J.</b>,
-     <i class='BibTitle'>Semigroups, A package for semigroups and monoids,
-             Version 3.0.20</i>
-     (<span class='BibYear'>2018</span>)<br />
-    (<span class='BibNote'>GAP package</span>),
-    <span class='BibHowpublished'><a href="https://gap-packages.github.io/Semigroups">https://gap-packages.github.io/Semigroups</a></span>.
+    [<span class='BibKey'>Mit18</span>]   <b class='BibAuthor'>J. D. Mitchell et al,</b>,
```

Comment:
(I'm not sure whether `Mit18` is James's preferred cite key.)

Reply:
I'm not sure either.  There currently isn't one, so I just went for the default.

Reply:
Okay :) My guess would be 'no bib key'... but we'll see!